src/block/CustomiesBlockFactory.php

```diff
@@ -175,7 +176,7 @@ public function registerBlock(Closure $blockFunc, string $identifier, ?Model $mo
 				->setString("group", $creativeInfo->getGroup() ?? ""))
 			->setInt("molangVersion", 1);
 
-		CreativeInventory::getInstance()->add($block->asItem());
+		CreativeInventory::getInstance()->add($block->asItem(), CreativeCategory::CONSTRUCTION); // TODO: not hardcode this
```

Comment:
I leave these here for review - Not sure how to deal with this without breaking compatiblity

Reply:
> Why not use $creativeInfo?

They are different (Customies one vs PMs one)

Reply:
You can use a match statement to convert the string to a pm enum